### DISCUSSION TITLE
Ubuntu Desktop deployments

### DISCUSSION THREAD

**Discussion by redfishstix** [vmware/packer-examples-for-vsphere#812] [Help] [answered]
Hi,
I was wondering how packer is working with the ubuntu desktop deployments. Every time I try to use packer, ubuntu will boot into a GUI install and packer fails after a few minutes. Have you ever tested this? I'm wondering if i'm missing something. Any idea? Thanks!

**Comment by tenthirtyam** [comment, marked as answer]
For this project Ubuntu Desktop images have not been planned, tested, or added.
